experiment/scheduler.py

```diff
@@ -652,12 +652,34 @@ def update_started_trials(trial_proxies, trial_id_mapping, core_allocation):
     return started_trials
 
 
+def get_corpus_variant_id_mapping(trials):
```

Comment:
Not sure if I understand... maybe you can clarify?

It sounds like you are proposing that groups of M trials for each fuzzer share a starting corpus, rather than just a single Nth trial sharing the corpus? So instead of

```
AFL  +  corpus 1
libfuzzer + corpus 1
AFL  +  corpus 2
libfuzzer + corpus 2
AFL  +  corpus 3
libfuzzer + corpus 3
...
AFL  +  corpus 50
libfuzzer + corpus 50
```

you might have this for e.g. 50%2:

```
AFL  +  corpus 1
libfuzzer + corpus 1
AFL  +  corpus 1
libfuzzer + corpus 1
AFL  +  corpus 2
libfuzzer + corpus 2
...
AFL  +  corpus 25
libfuzzer + corpus 25
```

If so, this seems reasonable to me, but maybe we add it in a future set of changes?

Reply:
Ah sure! I was just curious : )